### PR TITLE
feat(docs): add newline formatting for notes tip

### PR DESCRIPTION
**Pull request by ShaunSHamilton**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

This PR adds a mention of separating a Markdown note by a newline.

### CODE REVIEW

File: docs/how-to-work-on-coding-challenges.md [hunk 1]
```diff
@@ -276,8 +276,8 @@ The following is an example of code:
 ```
 ````
 
-- Additional information in the form of a note should be formatted `**Note:** Rest of note text...`
-- If multiple notes are needed, then list all of the notes in separate sentences using the format `**Notes:** First note text. Second note text.`.
+- Additional information in the form of a note should be separated by a newline, and formatted: `**Note:** Rest of note text...`
```
Comment (ojeytonwilliams) [inline docs/how-to-work-on-coding-challenges.md:279]:
```suggestion
- Additional information in the form of a note should be surrounded by blank lines, and formatted: `**Note:** Rest of note text...`
```
Hey Shaun, did you mean a note should be like this:

```
optional stuff before followed by an empty line

**Note:** Rest of note text...

optional stuff after preceded by an empty line
```

If so, what do you think to this wording?

Comment (ShaunSHamilton) [reply docs/how-to-work-on-coding-challenges.md:279]: That is clearer 👍

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 

Thank you for your pull request! We look forward to additional contributions!

**Review by moT01 — APPROVED** [review]
LGTM @ShaunSHamilton  🎉

**Pull request merged**